tests/queries/0_stateless/01455_time_zones.sql

```diff
@@ -0,0 +1,9 @@
+-- There are currently 562 timezones which are used from the binary embedded inside the ClickHouse binary
+-- Refer: contrib/cctz-cmake/CMakeLists.txt for the complete list. The count should match when queried.
+-- SELECT count(*)
+-- FROM system.time_zones
+--
+-- ┌─count()─┐
+-- │     562 │
+-- └─────────┘
+SELECT if ((SELECT count(*) FROM system.time_zones) = 562, 'ok', 'fail');
```

Comment:
In recent commit we started to generate the complete list on the fly. And the number of timezones may change. I will just check that there are more than 500 :)